src/qt/radiobut.cpp

```diff
@@ -45,17 +45,17 @@ bool wxRadioButton::Create( wxWindow *parent,
 
     if ( (style & wxRB_GROUP) || (style & wxRB_SINGLE) )
     {
-        m_qtButtonGroup = new QButtonGroup();
-        m_qtButtonGroup->addButton( m_qtRadioButton );
-
         if ( style & wxRB_SINGLE )
         {
             // Ensure that other buttons cannot join the last existing group
             m_lastGroup.erase(parent);
         }
         else
         {
-            m_lastGroup[parent] = m_qtButtonGroup;
+            QButtonGroup* qtButtonGroup = new QButtonGroup();
+            qtButtonGroup->addButton( m_qtRadioButton );
```

Comment:
No, these 2 lines must be moved back!
A button with `wxRB_SINGLE` still needs to be in its own group, alone. Otherwise it could be grouped with other radio buttons that are not part of an explicit group.

Reply:
Ah yes, my bad. Thanks for pointing that out.

Reply:
No problem.

This looks good to go now, I don't see other issues with it.